### PR TITLE
feat: uptake latest components DAH-1006

### PR DESCRIPTION
**Pull request by emilyjablonski**
Uptakes the latest components which includes the SiteHeader and ListingCard A11Y updates

I'm not sure of a great solution for maintaining the react query without creating another list of all the URLs we do want to be react-ified, so I haven't done that here

### CODE REVIEW

**Comment by akegan** [comment]
@emilyjablonski is this tied to a ticket?

File: app/javascript/layouts/Layout.tsx [hunk 1]
```diff
@@ -157,7 +157,7 @@ const Layout = (props: LayoutProps) => {
           flattenSubMenus={true}
           imageOnly={true}
           mobileText={true}
-          logoWidth={"medium"}
+          logoWidth={"base"}
```
Comment (akegan) [inline app/javascript/layouts/Layout.tsx:160]: the logo looks worse on this review app than on full. I don't think we need to change it?
This app:
image
Full:

**Comment by emilyjablonski** [comment]
@akegan There is a problem on DAHLIA production currently that I think will require the logo to be smaller, we can touch base offline

**Comment by akegan** [comment]
review app build is failing for this one. Sounds like you maybe need to run yarn install again?
```
-----> Building on the Heroku-18 stack
-----> Using buildpacks:
       1. https://github.com/heroku/heroku-buildpack-nodejs
       2. https://github.com/heroku/heroku-buildpack-ruby
       3. https://github.com/bobbus/image-optim-buildpack
-----> Node.js app detected
       
-----> Creating runtime environment
       
       NPM_CONFIG_LOGLEVEL=error
       USE_YARN_CACHE=true
       NODE_ENV=production
       NODE_MODULES_CACHE=true
       NODE_VERBOSE=false
       
-----> Installing binaries
       engines.node (package.json):  unspecified
       engines.npm (package.json):   5.3.0
       engines.yarn (package.json):  1.x.x
       
       Resolving node version 14.x...
       Downloading and installing node 14.18.2...
       Bootstrapping npm 5.3.0 (replacing 6.14.15)...
       npm 5.3.0 installed
       Resolving yarn version 1.x.x...
       Downloading and installing yarn (1.22.17)
       Installed yarn 1.22.17
       
-----> Installing dependencies
       Installing node modules (yarn.lock)
       yarn install v1.22.17
       info Merge conflict detected in yarn.lock and successfully merged.
       [1/4] Resolving packages...
       error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
       info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
-----> Build failed
 !     Outdated Yarn lockfile
       Your application contains a Yarn lockfile (yarn.lock) which does not
       match the dependencies in package.json. This can happen if you use npm
       to install or update a dependency instead of Yarn.
       Please run the following command in your application directory and check
       in the new yarn.lock file:
       $ yarn install
       $ git add yarn.lock
       $ git commit -m "Updated Yarn lockfile"
       $ git push heroku main
    
       https://help.heroku.com/TXYS53YJ
 !     Push rejected, failed to compile Node.js app.
 !     Push failed
```

File: yarn.lock [hunk 2]
```diff
@@ -17128,5 +17128,9 @@ zen-observable-ts@^1.0.0:
 
 zen-observable@0.8.15:
   version "0.8.15"
+<<<<<<< HEAD
```
Comment (akegan) [inline yarn.lock:17131]: @emilyjablonski actually looks like there's an unresolved merge conflict in the yarn.lock

Comment (akegan) [reply yarn.lock]: just fixed and pushed

**Review by akegan — APPROVED** [review]
I don't think this is a dealbreaker, but it looks like you can tab forward into the my account menu, but you can't shift+tab back to get to earlier links. 
image

I'll file an issue in the bloom project for it. 

Otherwise looks good.

**Pull request merged**